includes/reader-activation/class-reader-activation.php

```diff
@@ -1556,6 +1556,10 @@ public static function is_reader_verified( $user ) {
 			return null;
 		}
 
+		if ( defined( 'NEWSPACK_ALLOW_MY_ACCOUNT_ACCESS_WITHOUT_VERIFICATION' ) && NEWSPACK_ALLOW_MY_ACCOUNT_ACCESS_WITHOUT_VERIFICATION ) {
```

Comment:
Is this supposed to be included in this PR? Can't see it mentioned in the PR descriptions.

Reply:
Ah yes, sorry! I had introduced this constant in #2845 for convenience of testing (I got tired of having to do the verification step when I was testing unrelated functionality), but it turns out I missed another area where we checked for verification, so this plugs that hole.